protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -565,6 +565,13 @@ message GetOrderResponse {
     vega.Order order = 1;
 }
 
+message OrderFilter {
+    repeated vega.Order.Status statuses = 1;
+    repeated vega.Order.Type types = 2;
+    repeated vega.Order.TimeInForce time_in_forces = 3;
+    bool exclude_liquidity = 4;
+}
```

Comment:
Why not just have those at the top level of the ListOrderRequest? would make builing the query simpler + it's not reused anywhere